CMakeLists.txt

```diff
@@ -131,7 +131,9 @@ if (CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
     message( STATUS "nGraph-TensorFlow using CXX11 ABI:  ${NGRAPH_TF_CXX11_ABI}" )
 endif()
 
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -D_GLIBCXX_USE_CXX11_ABI=${NGRAPH_TF_CXX11_ABI}")
+if(DEFINED ${NGRAPH_TF_CX11_ABI})
```

Comment:
This shouldn't be a parameter. This should be auto detected like it's done now.

Reply:
The auto-detection (`set(NGRAPH_TF_CXX11_ABI "${TensorFlow_CXX_ABI}")) is only enabled for GNU compilers. For Clang, this will pass an empty flag to ```-D_GLIBCXX_USE_CXX11_ABI```

Reply:
As discussed, I'll move this to the if('GNU" above